### DISCUSSION TITLE
second iteration inside first one

### DISCUSSION THREAD

**Discussion by muzunov** [downforcetech/fluidxml.php#20] [Questions]
Hi Daniele. I as wondering how add second iteration inside first one, i red documentation but i cant do that. Like in your example, i  fetch data like author and book from mysql make iteration, but every book has a images stored in other table and i make another query. I try but..... errors :)

**Comment by daniele-orlando** [comment]
Sorry for the late reply but I was away.

If you could post an excerpt of your code I could understand what is not working in your specific case.
